text/0000-as-millis.md

```diff
@@ -0,0 +1,35 @@
+- Feature Name: as_millis
+- Start Date: 2016-03-17
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Add `as_millis` function to `std::time::Duration`.
+
+# Motivation
+[motivation]: #motivation
+
+Working with milliseconds is very common and since `std::time::Duration` has the `from_millis` and `as_secs` functions it 
+makes sense to create a `as_millis`.
+
+# Detailed design
+[design]: #detailed-design
+
+Add a `as_millis` function on `std::time::Duration` that divides `self.nanos` by `NANOS_PER_MILLI` and returns it.
```

Comment:
I think the usecase for the whole duration would be the main one. Another RFC should add a way to construct a subsec duration from the duration.